net/freeradius-server/Makefile

```diff
@@ -0,0 +1,607 @@
+#
+# Copyright (C) 2008-2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=freeradius-server
+PKG_VERSION:=3.0.11
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=\
+	ftp://ftp.freeradius.org/pub/freeradius/ \
+	ftp://ftp.freeradius.org/pub/freeradius/old/
+PKG_MD5SUM:=e8a748fe824223ed488cc74a1645023e
+PKG_MAINTAINER:=Daniel Golle <daniel@makrotopia.org>
```

Comment:
I would rather not.
I have bumped freeradius-server to v3.x since version 2.x has been marked as EOL and I didn't want the work to be wasted, however :
I have used freeradius with a limited set of modules (no SQL, no LDAP) in a specific context.
I would not be able to answer accurately any question related to this package and its modules.
I'm unlikely to use it again in the near futur.